scripts/lint-changed.sh

```diff
@@ -3,4 +3,3 @@
 MD_FILES=`git diff --cached --name-only | tr " " "\n" | egrep ^.*\.md$`
 
 # Execute Markdown lint if any markdown files have been changed and added to git
-[[ -z "$MD_FILES" ]] || GEM_PATH=.gem .gem/bin/mdl "$MD_FILES"
```

Comment:
@Qtrain  Actually, I don't mean that. You can have a look at the issue #299 and #300 
By the way, what I say 'remove the gem path' is just use the `mdl` comand directly.

Try this one:

    MD_FILES=`git diff --cached --name-only | tr " " "\n" | grep '^.*\.md$'`
    # Execute Markdown lint if any markdown files have been changed and added to git
    [[ -z "$MD_FILES" ]] || mdl "$MD_FILES"
    exit 0